components/Note/index.js

```diff
@@ -0,0 +1,113 @@
+import { useRouter } from "next/router";
+import React, { useState, useRef, useEffect } from "react";
+import styled from "styled-components";
+import useLocalStorageState from "use-local-storage-state";
+import { SlNote } from "react-icons/sl";
+import { TfiCheck } from "react-icons/tfi";
+
+const Note = ({ project }) => {
+  const router = useRouter();
+  const { id } = router.query;
+
+  const [currentNote, setCurrentNote] = useLocalStorageState(`note-${id}`, "");
+  const [isEditing, setIsEditing] = useState(false);
```

Comment:
Just a thought about the naming of the usestate:  maybe consider naming it [isEditMode, setIsEditMode]  for better understanding :)

Reply:
you are right, it is indeed a better naming